tests/components/updater/test_init.py

```diff
@@ -174,15 +215,37 @@ def test_new_version_shows_entity_after_hour_hassio(
 ):
     """Test if new entity is created if new version is available / hass.io."""
     mock_get_uuid.return_value = MOCK_HUUID
-    mock_get_newest_version.return_value = mock_coro((NEW_VERSION, ""))
+    mock_get_newest_version.return_value = mock_coro((NEW_VERSION, RELEASE_NOTES))
     mock_component(hass, "hassio")
     hass.data["hassio_hass_version"] = "999.0"
 
     res = yield from async_setup_component(hass, updater.DOMAIN, {updater.DOMAIN: {}})
     assert res, "Updater failed to set up"
 
+    yield from hass.async_block_till_done()
+
     with patch("homeassistant.components.updater.current_version", MOCK_VERSION):
         async_fire_time_changed(hass, dt_util.utcnow() + timedelta(hours=1))
         yield from hass.async_block_till_done()
 
-    assert hass.states.is_state(updater.ENTITY_ID, "999.0")
+    assert hass.states.is_state("binary_sensor.updater", "on")
+    assert (
+        hass.states.get("binary_sensor.updater").attributes["newest_version"] == "999.0"
+    )
+    assert (
+        hass.states.get("binary_sensor.updater").attributes["release_notes"]
+        == RELEASE_NOTES
+    )
+
+
+@asyncio.coroutine
+def test_setup_dispatcher(hass, mock_dispatcher_connect):
+    """Test if we cleanly setup and close the dispatcher."""
+    mock_get_uuid.return_value = MOCK_HUUID
+    mock_get_newest_version.return_value = mock_coro((NEW_VERSION, RELEASE_NOTES))
+
+    res = yield from async_setup_component(hass, updater.DOMAIN, {updater.DOMAIN: {}})
+    assert res, "Updater failed to set up"
+
+    yield from hass.async_block_till_done()
+    assert mock_dispatcher_connect.called
```

Comment:
We don't need to test this. We know that this is working since the entity state is updated correctly in the other tests.

Reply:
removed